src/Umbraco.Infrastructure/EmailSender.cs

```diff
@@ -75,35 +31,33 @@ public void Send(EmailMessage message)
         /// <returns></returns>
         public async Task SendAsync(EmailMessage message)
         {
-            if (_smtpConfigured.Value == false && _enableEvents)
+            if (_globalSettings.IsSmtpServerConfigured == false)
             {
-                OnSendEmail(new SendEmailEventArgs(message));
+                return;
             }
-            else if (_smtpConfigured.Value == true)
-            {
-                using (var client = new SmtpClient())
-                {
-                    await client.ConnectAsync(_globalSettings.Smtp.Host,
-                        _globalSettings.Smtp.Port,
-                        (MailKit.Security.SecureSocketOptions)(int)_globalSettings.Smtp.SecureSocketOptions);
 
-                    if (!(_globalSettings.Smtp.Username is null && _globalSettings.Smtp.Password is null))
-                    {
-                        await client.AuthenticateAsync(_globalSettings.Smtp.Username, _globalSettings.Smtp.Password);
-                    }
+            using (var client = new SmtpClient())
+            {
+                await client.ConnectAsync(_globalSettings.Smtp.Host,
+                    _globalSettings.Smtp.Port,
+                    (MailKit.Security.SecureSocketOptions)(int)_globalSettings.Smtp.SecureSocketOptions);
 
-                    var mailMessage = ConstructEmailMessage(message);
-                    if (_globalSettings.Smtp.DeliveryMethod == SmtpDeliveryMethod.Network)
-                    {
-                        await client.SendAsync(mailMessage);
-                    }
-                    else
-                    {
-                        client.Send(mailMessage);
-                    }
+                if (!(_globalSettings.Smtp.Username is null && _globalSettings.Smtp.Password is null))
+                {
+                    await client.AuthenticateAsync(_globalSettings.Smtp.Username, _globalSettings.Smtp.Password);
+                }
 
-                    await client.DisconnectAsync(true);
+                var mailMessage = ConstructEmailMessage(message);
+                if (_globalSettings.Smtp.DeliveryMethod == SmtpDeliveryMethod.Network)
+                {
+                    await client.SendAsync(mailMessage);
+                }
+                else
+                {
+                    await client.SendAsync(mailMessage);
```

Comment:
This seems odd.. I guess there was a reason it was not async in some cases before

Reply:
It has been like that since: https://github.com/umbraco/Umbraco-CMS/commit/4b7bd6482efd999588ccac27c174d8ab6c6de3b2, where it was copied from another file..

Reply:
Hmm... one might argue that there is no reason for the if/else statement 😆 silly me. That one slipped my own review. On the other hand I guess there is no problem leaving the synchronous email send be for now, just to play it safe?

Reply:
I agree.. Play safe and we can merge

Reply:
@bergmania it's updated now 👍

Reply:
Now we have another conflicts 🙈

Reply:
Ha! I'll resolve 😆

Reply:
That ought to do it 😄